mvp-spec/dsp-implementation.md

```diff
@@ -196,138 +166,62 @@ are subject to adaptation to the context. For instance, if the
-<!--partial-end-->
+## The Identity endpoint
 
-#### Signatures
+The Contracting Party must expose an Identity endpoint. It gathers information for verifying the identity and the transmissions that it signed.
 
-Note that the signature of the `source` in the Transmission Request is 
-generated with the following string:
-
-<!--partial-begin { "files": [ "transmission-request-signature-string.txt" ], "block": "" } -->
-<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+It is reachable at the following endpoint:
 ```
-transmission_request_receiver_domain        + '\u2063' +
-transmission_request.source.domain          + '\u2063' + 
-transmission_request.source.timestamp       + '\u2063' + 
-seed.source.signature
+GET https://<domain>/prebid/v1/identity
 ```
-<!--partial-end-->
 
-The signatures of the `source`s in the Transmission Results are generated with
-the following string:
+| Message  | Format|
+|----------|-------|
+| Request  | [get-identity-request.md](./model/get-identity-request.md)  |
+| Response | [get-identity-response.md](./model/get-identity-response.md)|
 
-<!--partial-begin { "files": [ "transmission-result-signature-string.txt" ], "block": "" } -->
-<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+<details>
+<summary>Identity Response Example</summary>
+
+Request:
+```http
+GET /v1/identity
+Host: my-dsp.io
 ```
-transmission_result.receiver                + '\u2063' +
-transmission_result.status                  + '\u2063' 
-transmission_result.source.domain           + '\u2063' + 
-transmission_result.source.timestamp        + '\u2063' + 
-seed.source.signature      // -> The Seed associated to the given Transaction Result
+
+Response:
+```json
+{
+  "name": "My DSP",
+  "keys": [
+    {
+      "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEEiZIRhGxNdfG4l6LuY2Qfjyf60R0\njmcW7W3x9wvlX4YXqJUQKR2c0lveqVDj4hwO0kTZDuNRUhgxk4irwV3fzw==\n-----END PUBLIC KEY-----",
+      "start": 1641034200,
+      "end": 1646132400
+    }
+  ],
+  "type": "vendor",
+  "version": "0.1"
+}
 ```
-<!--partial-end-->
+</details>
 
-⚠️ **It is not required to verify those signatures in the scope of the MVP.**
-The details of those signatures are here just for the understanding purpose.
+## Standalone implementation of the Transmissions protocol
 
-#### Example of a Transmission Request
+If the Sender and the Receiver want to integrate the Transmissions in an existing
+communication, they must integrate the Standalone version in of the Transmission Protocol
+in their existing models. 
 
+| Message                          | Format|
+|----------------------------------|-------|
+| Standalone Transmission Request  | [transmission-requests-standalone.md](./model/transmission-requests-standalone.md)  |
```

Comment:
In an existing communication, for one HTTP request/response (e.g between a SSP and a DSP) you can have many transmission requests/responses.

BTW, why singular "model" and plural "json-schemas"  for the directory names?

Reply:
Ok thanks.

> BTW, why singular "model" and plural "json-schemas" for the directory names?

It felt natural when I created the directories, but now I guess it still makes sense that there are multiple schemas, but the different `.md` files constitute a single model. There are not multiple models.

Reply:
Ok, thank you. So you mean a data model, I guess. Usually, a data model is used to represent a domain, a DB representation, or an abstract concept. Here we describe API and data exchanges, so I thought it would mean data structures/models in a sense of programming. It is out of the scope of this PR and may not be significant but maybe a more accurate name would be "API" or even "data-formats" like it is expressed in json-schema.org.